wsagent/che-core-api-git-shared/src/main/java/org/eclipse/che/api/git/shared/LogRequest.java

```diff
@@ -21,21 +21,96 @@
  */
 @DTO
 public interface LogRequest extends GitRequest {
-    /** Filter revisions list by range of files. */
+
+    /** @return revision range since */
+    String getRevisionRangeSince();
+
+    /** Set revision range since */
+    void setRevisionRangeSince(String revisionRangeSince);
+
+    /**
+     * Create a {@link LogRequest} object based on a given revision range since.
+     *
+     * @param revisionRangeSince
+     *         revision range since
+     * @return a {@link LogRequest} object
```

Comment:
I mean that `@return a {@link LogRequest} object` tells nothing except methods return type.
As for me it would be better to remove such fields only in this class, or try to provide more information there.